src/app/dataScripts/fetchSpendingOverTimeData.ts

```diff
@@ -21,16 +22,29 @@ export default async function fetchSpendingOverTimeData() {
         await mkdir(dataDir);
     }
 
-    let STATE_CODES: string[];
     if (!existsSync(statesJSON)) {
         console.warn('Exiting because states json does not exist');
         return;
-    } else {
-        const states = await fs.readFile(statesJSON);
-        STATE_CODES = JSON.parse(states.toString()).map(
-            (state: State) => state.code
-        );
     }
+    if (!existsSync(perCapitaJSON)) {
+        console.warn('Exiting because per capita json does not exist');
+        return;
+    }
+
+    const states = await fs.readFile(statesJSON);
+    const STATE_CODES: string[] = JSON.parse(states.toString()).map(
+        (state: State) => state.code
+    );
+    const perCapitaData = await fs.readFile(perCapitaJSON);
+    const perCapitaDataParsed = JSON.parse(perCapitaData.toString());
+    const STATE_POPULATIONS: Map<string, number> = new Map(
```

Comment:
Any reason some of these vars are SCREAMING_SNAKE_CASE while others are camelCase?

Reply:
nope! Just aligning with what was already here, but I'll camel case it.